site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -0,0 +1,199 @@
+---
+menu: Active Proposals
+name: Interest Invokers (Explainer)
+layout: ../../layouts/ComponentLayout.astro
+---
+
+- Authors: [Keith Cirkel](https://github.com/keithamus)
+
+{/* START doctoc generated TOC please keep comment here to allow auto update */}
+{/* DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE */}
+
+# Interest Invokers
+
+## Pitch in Code
+
+```html
+<button interesttarget="my-tooltip">Hover/Focus to show the tooltip</button>
+
+<span popover=hint id="my-toolip">This is the tooltip</span>
+```
+
+## Introduction
+
+Following [invokers](../invokers.explainer), adding an `interesttarget`
+attribute to interactive elements: starting with `<button>`, `<input
+type="button">`/`<input type="reset">`, and `<a>` (perhaps expanding to
+`<area>`, `<input>`, `<textarea>`, `<select>`, `<summary>` or maybe more, see
+[#14](https://github.com/openui/open-ui/issues/908)). This would allow
+disclosure of high fidelity tooltips in a more accessible and declarative way.
+Elements with `interesttarget` will - when hovered, long pressed, or focussed -
+dispatch an `InterestEvent` on the element referenced by `interesttarget`, with
+some default behaviours.
+
+### Terms
+
+- Interest/Shows Interest: The action of _Interest_ refers to the user "landing"
+  on an element without invoking it, using a HID. If the HID is a mouse, this
+  would be a `hover` event. If the HID is a touch screen, this would be a long
+  press on the element using a stylus or finger, if the HID is a keyboard, this
+  would be when the element is focussed. For other HIDs such as eye tracking or
```

Comment:
Not sure if it's overly relevant but eye tracking causing interest events would be a potentially big privacy risk. This is why vision pro afaik explicitly doesn't trigger hover based on eye tracking. Interest would be the same.

Potentially worth calling out the risks or just removing references to eye tracking full stop?

Reply:
...or adding something like "eye tracking (potentially combined with another form of explicit confirmation such as a finger-tap to avoid privacy issues)..."? This, again, is where it's up to the UA to decide what constitutes "interest". The common examples of hovering with a mouse or keyboard-focusing should be very clear cases of interest, but others need UA help.